README.md

```diff
@@ -87,8 +65,9 @@ For an in-depth tutorial: http://marcelog.github.com/articles/pami_introduction_
 
 ```
 $options = array(
-    'log4php.properties' =>
-         RESOURCES_DIR . DIRECTORY_SEPARATOR . 'log4php.properties',
+    'psr3_logger' => $log,
```

Comment:
`$log` is not defined here, and no default logger is provided.

Suggestion: add something along the lines of

```
$log = new \Psr\Log\NullLogger();
```